examples/formbot/domain.yml

```diff
@@ -89,33 +89,35 @@ actions:
 
 forms:
   restaurant_form:
-    cuisine:
-    - type: from_entity
-      entity: cuisine
-      not_intent: chitchat
-    num_people:
-    - type: from_entity
-      entity: number
-      intent: [inform, request_restaurant]
-    outdoor_seating:
-    - type: from_entity
-      entity: seating
-    - type: from_intent
-      intent: affirm
-      value: true
-    - type: from_intent
-      intent: deny
-      value: false
-    preferences:
-    - type: from_intent
-      intent: deny
-      value: no additional preferences
-    - type: from_text
-      not_intent: affirm
-    feedback:
-    - type: from_entity
-      entity: feedback
-    - type: from_text
+    ignored_intents: [chitchat, inform]
```

Comment:
ok to this for `chitchat`, but I think `inform` breaks most of the bot. Also if you ignore `chitchat` then you don't need to ignore it again in line 97